deploy/crds/aws_v1alpha1_awsfederatedrole_crd.yaml

```diff
@@ -49,13 +49,23 @@ spec:
                         items:
                           type: string
                         type: array
+                      condition:
+                        properties:
+                          StringEquals:
+                            additionalProperties:
+                              type: string
+                            description: A map of the condition
+                            type: object
+                        type: object
                       effect:
                         type: string
                       principal:
                         properties:
                           AWS:
                             description: aws account id
-                            type: string
+                            items:
+                              type: string
+                            type: array
```

Comment:
Should this field be an array of strings?

Reply:
Answered by: https://github.com/openshift/aws-account-operator/blob/master/pkg/apis/aws/v1alpha1/awsfederatedrole_types.go